tutorial1_intro/demo/main.js

```diff
@@ -34,5 +34,7 @@ d3.csv("../../data/surveyResults.csv").then(data => {
     .selectAll("td")
     .data(d => Object.values(d))
     .join("td")
+    // update the below logic to apply to your dataset
+    .attr("class", d => +d > 3 ? 'high' : null)
```

Comment:
we probably want to add a corresponding style to this also so that when they load the demo they see a difference in those cells